documentation/authentication.md

```diff
@@ -0,0 +1,135 @@
+Authentication
```

Comment:
I would suggest replacing all references to "API Token" with "API Key" since we aren't using tokens.